components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -494,6 +499,33 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         return user;
     }
 
+    public async supportConfigureWorkspaceTimeout(ctx: TraceContext): Promise<boolean> {
```

Comment:
The `support` puts me off a bit: We don't use it, yet, anywhere.

Why not stick with `can` or `may`? :thinking: 

Also, we happen to make (nearly) exactly the same method over [here](https://github.com/gitpod-io/gitpod/blob/b20150fb3c2f3cd7e8af8e961a0c1a1b1f5ba786/components/server/ee/src/workspace/gitpod-server-impl.ts#L497-L502). Maybe it's worth combining the two? :shrug:

Reply:
> the method signature is not same

The signature is effectively identical: If we make `maySetTimeout` public, it's basically `maySetWorkspaceTimeout(ctx: TraceContext)`, where we get the `user` on the first or second line.

> and it is only in enterprise

I agree that's super confusing, and we should change that very soon: it' the code we use for all distribution modes: there is :zero: difference, we ship that "ee" code to everyone.